queries/rego/highlights.scm

```diff
@@ -0,0 +1,14 @@
+; highlights.scm
+"import" @include
+"package" @include
+
+
+(reserved_keywords) @keyword
+(comment) @comment
```

Comment:
you could also add the comment parser to the injections, like in https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/go/injections.scm

Reply:
Done. 
But can you please, help me understand, why that file is needed? 
As I understood, injections are a part of external bindings as C, but there is none

Reply:
you can check our docs about this at https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md#injections. Injections are done from the client using tree-sitter, not from the parser.